docs/app-dev/abci-cli.md

```diff
@@ -285,9 +285,6 @@ In another window, start the `abci-cli console`:
 
 ```sh
 > set_option serial on
```

Comment:
I think you need to remove this line as well. It looks like there's a few places further up where we mention `setOption`. I don't think this is necessary because in https://github.com/tendermint/tendermint/pull/9143 I am removing the counter app entirely